FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -741,24 +742,65 @@ - (void)signInWithEmail:(NSString *)email
       [[FIRVerifyPasswordRequest alloc] initWithEmail:email
                                              password:password
                                  requestConfiguration:_requestConfiguration];
-
   if (![request.password length]) {
     callback(nil, [FIRAuthErrorUtils wrongPasswordErrorWithMessage:nil]);
     return;
   }
-  [FIRAuthBackend
-      verifyPassword:request
-            callback:^(FIRVerifyPasswordResponse *_Nullable response, NSError *_Nullable error) {
-              if (error) {
+    if (YES) { // iOS 14.0+
```

Comment:
@renkelvin making sure I followed this logic correctly.

We start by fetching a recaptcha token (with ForceRefresh false) and pass the response to verifyPassword. If verifyPassword retuns MISSING_RECAPTCHA_TOKEN error, then we retry tokenFetch with ForceRefresh true?

Assuming my understanding is right, I think we also need to handle INVALID_RECAPTCHA_TOKEN error this way.